documentation/concepts/notifications/notification-concept.adoc

```diff
@@ -57,6 +58,11 @@ A task has been deferred::
 
 The task has been deferred, either in the UI, a notification or "remotely" by a sync adapter.
 
+The device boots up::
```

Comment:
The intent is good, but this is the wrong place IMO. This section is about task properties which may result in a notification, i.e. for a notification to show up on boot it still requires one of these triggers having "occurred" either before shut down or during off-time. The boot event alone is not sufficient for a notification be created.

Restoring notifications on boot deserves its own section. Also, we should not just show a notification for all tasks which have started in the past. Some of these may have been acknowledged before the device was shut down. It could be annoying if they show up again on every reboot. So we need to maintain a state of visible notifications (hence the state diagram).

Reply:
I am sorry, then I understood this wrongly. I thought you would collect all events that might case an notification to be fired. Of course a check needs to be carried out in order to avoid reissuing the same events over and over.
This is also the reason for the first change in the file.